### DISCUSSION TITLE
v1.1.15

### DISCUSSION THREAD

**Discussion by unocelli** [frangoteam/FUXA#928] [Releases]
New features:
- Add 3-5 year DAQ storage retention option
- Settings to hide chart toolbar in history mode (#913)
- ENTER-Key event of Input control, to send input value as parameter to a script (#891)
- Graph Pie control
- Script function $getTagId to get Tag ID from name and $enableDevice to enable and disable Device
- Add image control with link
- Settings to enable authentication for editor only
- Script option to run only on start and with delay
- DateTime format in Tag options settings
- Int64 type to modbus Tags
- German translation (#894 Thanks @EtherGraf)
- Configurable header items: button, label and icons (#437)
- Settings to improve the performance of the transmission of tag values, depending on the view, default enabled (#768 #872)
- ACK All Alarms button to the header of alarms view
- Dialog to modify all the Tags present in the settings of selected element

Fixed:
- Save settings of alarm retention
- Gauge range settings (#922)
- ACK all alarms of floating (cause wrong alarms counter)
- Loop rotation animation (#914)
- String value in Input control
- Authorization of report download (#902)
- OPCUA connection error: End point must exist (#901 Thanks @zsdycs)
- Image resources not found in dev mode (#897 Thanks @k1nz)
- Date range query of report alarms history
- API setTagValue error result (#864)
- Undefined tag name in daqstore for influxdb2 (#889 Thanks @mauroalexandre)
- Date column, string value, full value row, sorting of table control in history mode (#764)
- Check Group/Ungroup and Open settings of selected element in item list (svg element)

<hr /><em>This discussion was created from the release <a href='https://github.com/frangoteam/FUXA/releases/tag/v1.1.15'>v1.1.15</a>.</em>

**Comment by robbudge** [comment]
Great Work
